config_test.go

```diff
@@ -238,28 +238,36 @@ func TestRetrieveRootPath(t *testing.T) {
 	type testData struct {
 		RawURL   string
 		FinalURL string
+		isError  bool
 	}
 
 	td := []testData{
-		{"mydomain.com/", "mydomain.com"},
-		{"mydomain.com?id=1", "mydomain.com"},
-		{"192.168.10.12/xxxxx", "192.168.10.12"},
-		{"mydomain.com/xxxxx?id=1", "mydomain.com"},
-		{"www.mydomain.com/xxxxx", "www.mydomain.com"},
-		{"http://www.mydomain.com/xxxxx", "http://www.mydomain.com"},
-		{"https://www.mydomain.com/xxxxx", "https://www.mydomain.com"},
-		{"https://www.mydomain.com/xxxxx?id=1", "https://www.mydomain.com"},
+		// :1234 is consided invalid url since the root url is also used as a
+		// hyperlink reference on the frontend.
+		{":1234", "", true},
```

Comment:
This should not be a valid URL since politeia is an external system. For use on other applications, add a host/scheme would work.